src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -7,6 +7,7 @@ import {Context} from '../../../context';
 import {getChatName} from '../../../utils/string/StringUtils';
 import {requestExternalStoragePermission} from '../../../utils/permission';
 import {uploadFile} from '../../../service/file';
+import ImageUtils from '../../../utils/image';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ImageUtils` is a good change as it seems to be replacing the old `uploadFile` function with a more specific `uploadImage` function. This could potentially improve readability and maintainability.
```diff
- import {uploadFile} from '../../../service/file';
+ import ImageUtils from '../../../utils/image';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
Removed the `import {uploadFile} from '../../../service/file';` because unused now